tasks/README.md

```diff
@@ -162,6 +162,17 @@ Add a webhook to your GitHub project on the Settings → Webhooks page of your p
 See [GitHub development documentation](https://developer.github.com/webhooks/)
 for more information.
 
+### Set up bot as collaborator
+
+ * On your project's "Settings → Manage Access" page, invite the [cockpituous user](https://github.com/cockpituous) as a collaborator.
+ * Once you see the "Pending invite", in the list, the clipboard symbol copies the corresponding `/invitations` URL to the clipboard. Send that to Martin Pitt (`pitti` in `#cockpit` on FreeNode IRC) to log into GitHub as cockpituous user and accept the invite.
```

Comment:
The way the last sentence here is phrased makes it sound like the reader will log into GH as cockpituous. Wonder if something like 

`Send that to Martin Pitt (...), who will accept the invite as cockpituous.`

Reply:
Maybe this would be worth to make the bus factor a bit bigger :)

Reply:
Good points you both. I added Marius, and fixed the sentence structure. I think Lars can do that as well, but he's on PTO so can't confirm right now.